spec/requests/api/v2/access_request_spec.rb

```diff
@@ -24,6 +24,115 @@
 
   subject { response }
 
+  describe 'GET #list' do
+    let(:list_access_requests_route) do
+      get "/api/v2/access_requests/#{admin_user.id}/list",
```

Comment:
Once you make the change as Dan's suggested, this will change to `get "/api/v2/access_requests"`. You don't want the admin user ID in there as they're unnecessary, the admin user details are already in the `Authorisation` header.

Reply:
Done. Thanks for the feedback.